.github/scripts/bump-nightly-version.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+##
+# @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+##
+
+set -euxo pipefail
+
+DATE=$(date '+%Y%m%d')
```

Comment:
(i wrote a comment but it disappeared 😢 )

since this publishes in the morning with today's date.. it does mean that commits that land on a day (say December 9th) are not in the `1209` release.

```sh
date --date="yesterday" '+%Y%m%d'
```

fixes that.


also this --date flag for date is in gnu date but not bsd/mac date.  so doesnt work on mac but should be fine on CI

![image](https://user-images.githubusercontent.com/39191/101708671-9b23cf80-3a42-11eb-8472-6b30b6d8d5aa.png)